### DISCUSSION TITLE
Feature suggestion - edit and commit multiple files through Github web

### DISCUSSION THREAD

**Discussion by PranayShah** [github/india#89] [GitHub Platform] [answered]
Extremely convenient to commit a single file through the editor on Github. AFAIK, only a single file can be committed as such. Many times we have to make changes across a bunch of files which do not even require code to be checked out on a system. Would really help there. 
Github supports resolving conflicts across multiple files and committing the merge, so the capability is already there. Hope it can be extended for this general use case. @shashban

**Comment by shashban** [comment, marked as answer]
Thanks for the suggestion @PranayShah 
Editing using GitHub UI is recommended for simple changes (typically a single file). If there are multiple files, you can create a branch, make all the changes and run validations in a PR.

**Reply by rohitnb** [reply]
We also have Codespaces on beta that allow you to spin up a VS Code on the browser with the repo already checked out. This could help you in these situations where you want to make quick edits via your browser. For more info on Codespaces, check [this doc](https://github.com/features/codespaces) out!
